local_planner/CMakeLists.txt

```diff
@@ -199,13 +199,14 @@ if(CATKIN_ENABLE_TESTING)
 	catkin_add_gtest(${PROJECT_NAME}-test test/main.cpp
 	                                      test/test_example.cpp
 	                                      test/test_common.cpp
-	                                      test/test_local_planner.cpp)
+	                                      test/test_local_planner.cpp
+	                                      test/test_planner_functions.cpp)
```

Comment:
This needs a rebase, there are new tests in `test_planner_functions.cpp` in master now.